### PR TITLE
Make "fixlengths" command ignore trailing empty fields in automatic m…

### PR DESCRIPTION
**Pull request by kyegupov**
…ode.

### CODE REVIEW

File: tests/test_fixlengths.rs [hunk 1]
```diff
@@ -27,6 +35,43 @@ fn prop_fixlengths_all_maxlen() {
 }
 
 #[test]
+fn prop_fixlengths_all_maxlen_trims() {
```
Comment (BurntSushi) [inline tests/test_fixlengths.rs]: Could you drop the `prop_` prefix on these test names? (These tests aren't using quickcheck.)

Comment (kyegupov) [inline tests/test_fixlengths.rs]: Done.

**Comment by kyegupov** [comment]
This was inspired by a public dataset (warning: big) https://s3.amazonaws.com/nyc-tlc/trip+data/yellow_tripdata_2016-07.csv

It is apparently a sloppy expert in the form of:
```
foo,bar,baz
a1,b1,c1,,
a2,b2,c2,,
```

**Comment by BurntSushi** [comment]
I squashed and merged this in https://github.com/BurntSushi/xsv/commit/64b49169f8d4368ad07062a80ee401be8a65004f

**Pull request closed without merging**